api/admin/package-lock.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "circulation",
```

Comment:
We should be able to totally drop this file. We don't need a `package-lock.json` checked in here now that we are pulling the admin interface in from a CDN.

Reply:
I can't do that yet because the [PR for pulling from CDN](https://github.com/ThePalaceProject/circulation/pull/15) has not yet been merged.

Reply:
🤦 My bad. I was sure that went in already. I guess I was thinking of the CDN PR for library-registry.